service/models.py

```diff
@@ -37,7 +37,8 @@ def __repr__(self):
 
     def serialize(self):
         """ Serializes a Supplier into a dictionary """
-        return {"supplierName": self.supplierName,
+        return {"id": str(self.id),
```

Comment:
The change is fine, I just want to point out that we can simply use supplier.to_json() and supplier.get_json() instead of serialize() and deserialze()...

Reply:
I agree with the approach you mentioned. The first time I saw you introduced the serialize and deserialize functions I was kinda lost lol. Because serialisation is not very "NoSQL" in my opinion as MongoDB itself is a sort of JSON document.

Anyway, thanks a lot for the suggestions. I have made the changes you asked about, please take a look and let me know if there is anything else that I can help :D